cavityDisplayGUI.py

```diff
@@ -111,11 +111,19 @@ def __init__(self, parent=None, args=None):
                     # .add_callback is called when descriptionPV changes value
                     descriptionPV.add_callback(partial(self.descriptionCallback, cavityWidget, descriptionPV))
 
-                    rfStatePV.add_callback(partial(self.rfStatusCallback, cavityWidget))
+                    # .add_callback is called when rfStatePV changes value
+                    rfStatePV.add_callback(partial(self.rfStatusCallback, rfStatusBar))
 
-    @staticmethod
-    def rfStatusCallback(cavityWidget: CavityWidget, value: int, **kw):
-        cavityWidget.underline = True if value == 1 else False
+    # Underlines cavity if RF is on
+    def rfStatusCallback(self, rf_StatusBar, value: int, **kw):
```

Comment:
I think this should still have the `@staticmethod` decorator since we don't actually use `self` anywhere haha

Reply:
Made the change!